server/db/index.js

```diff
@@ -1,21 +1,15 @@
 const mongoose = require('mongoose');
 
-exports.connect = (URI, cb) => new Promise((resolve, reject) => {
+exports.connect = async (URI) => {
   try {
-    mongoose.connect(URI, {
-      useNewUrlParser: true,
-      useUnifiedTopology: true
-    }, (err, data) => {
-      if (cb) {
-        resolve(cb(err, data));
-      } else {
-        if(err) reject(err);
-        resolve(data);
-      }
-    });
+    const res = await mongoose.connect(URI, { useNewUrlParser: true, useUnifiedTopology: true});
+    
+    if (!res) throw new Error();
+
+    console.log('Connected to Mongo Atlas');
```

Comment:
Nice refactor!!

Reply:
Yes, I used Atlas to get started on the crud operations more quickly. Latest PR now has Docker setup to build a Mongo container. Thank you  for the feedback!

Reply:
Also, I gave your write access so let me know if you are now able to merge to master.